csrankings.csv

```diff
@@ -15997,6 +16002,7 @@ Yücel Yemez,Koç University,http://network.ku.edu.tr/~yyemez,12csu04AAAAJ
 Z. Cihan Taysi,Yıldız Technical University,https://www.ce.yildiz.edu.tr/personal/cihan?lang=en,NOSCHOLARPAGE
 Z. Meral Özsoyoglu,Case Western Reserve University,http://engineering.case.edu/eecs/faculty-staff,QRrrUU0AAAAJ
 Zac Friggstad,University of Alberta,https://webdocs.cs.ualberta.ca/~zacharyf,NOSCHOLARPAGE
+Zachary Chase Lipton,Carnegie Mellon University,http://zacklipton.com/,MN9Kfg8AAAAJ

```

Comment:
Not in the directory.
https://www.cs.cmu.edu/directory

Reply:
SCS directory is not complete or frequently updated to be current. Different departments maintain their own frequently updated personnel lists. See Lipton's affiliation [here](https://www.ml.cmu.edu/people/affiliated-faculty.html) Also see his two MLD advisees [here](https://www.ml.cmu.edu/people/phd-students.html). He also advises some students in LTI, SCS afaik.